### PR TITLE
Refonte TPs GCC! Partie nouvel outil

### PR DESCRIPTION
**Pull request by Gregoire-perso**
Vous trouverez dans cette PR la partie concernant les changements propres à Hugo (principalement le thème et l'architecture nécessaire aux menus)

### CODE REVIEW

**Review by kolowy — APPROVED** [review]

**Review by amelie311098 — CHANGES_REQUESTED** [review]
Quelques typo
Des fichiers qui sont supprimés qui le devraient peut être pas
J'ai pas encore run en local pour vérifier que tout fonctionne bien, j'attendrai la 2e PR

File: README.md [hunk 1]
```diff
@@ -1,45 +1,96 @@
...
-Attribution-NonCommercial-ShareAlike 4.0 International
-License](http://creativecommons.org/licenses/by-nc-sa/4.0/).
-
 Ce dépôt contient tous les travaux pratiques [des stages Girls Can 
-Code!](https://gcc.prologin.org/). Les stages se concentrent autour de 
+Code!](https://girlscancode.fr/). Les stages se concentrent autour de 
 l'apprentissage du Python mais couvrent de nombreux sujets comme le réseau,
 la programmation web (html, css), unix ou encore la programmation orientée
-objet.
+objet.
```
[the remark this answers is not in the document]
Comment (Gregoire-perso) [reply README.md:6]: Je l'ai probablement ajouté par réflexe sans y faire attention

File: README.md [hunk 2]
```diff
@@ -1,45 +1,96 @@
...
+
+Il ne reste plus qu'a écrire le sujet dans le fichier `index.md`, ainsi
+que d'écrire un `README.md` décrivant rapidement le sujet. Ce fichier sera
+essentiellement destiné aux organisateurs et permet d'éclaicir certaines
+particularité du TP. 
+
+
+
+# Fonctionnement de Hugo
+Pour lancer le site en local, il suffit dhugo server --renderToDisk --noHTTPCachee faire
```
[the remark this answers is not in the document]
Comment (Gregoire-perso) [reply README.md:74]: Oops, va falloir que j'apprenne à utiliser Vim visiblement :P

File: 04_objet/ref.py [hunk 3]
```diff
@@ -1,65 +0,0 @@
-red = "\033[0;31m"
```
[the remark this answers is not in the document]
Comment (Gregoire-perso) [reply 04_objet/ref.py:1]: C'est un fail git ^^

File: 04_web_boni/04_web_boni.md [hunk 4]
```diff
@@ -1,111 +0,0 @@
----
```
[the remark this answers is not in the document]
Comment (Gregoire-perso) [reply 04_web_boni/04_web_boni.md:1]: Il a été renommé lui pourtant

File: 04_web_boni/04_web.md [hunk 5]
```diff
@@ -1,335 +0,0 @@
----
```
[the remark this answers is not in the document]
Comment (Gregoire-perso) [reply 04_web_boni/04_web.md:1]: Il était en doublons (c'est le sujet du `04_web` pas bonus)

File: 04_web_boni/source/background/css/style.css [hunk 6]
```diff
@@ -1,11 +0,0 @@
-html { /* Applique le fond "planche de bois" sur toute la page */
```
[the remark this answers is not in the document]
Comment (Gregoire-perso) [reply 04_web_boni/source/background/css/style.css:1]: Lui aussi c'est un doublons du `04_web` pas bonus

**Comment by Nhqml** [comment]
Plusieurs points :
- je suis d'accord avec Amélie, il y a peut-être des fichiers qui sont supprimés alors qu'ils ne devraient pas l'être ?
- cdn.mathjax.org n'existe plus (cf https://www.mathjax.org/cdn-shutting-down/) peut-être qu'il faudrait considérer changer de CDN ?
- la fonctionnalité de copie du code (qui est vraiment très cool btw !) utilise des fonctions dépréciées ([`document.queryCommandSupported()`](https://developer.mozilla.org/en-US/docs/Web/API/Document/queryCommandSupported) et [`document.execCommand()`](https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand)), ce qui veut dire que ça arrêtera de fonctionner un jour... Il faudrait peut-être considérer utiliser l'[API Clipboard](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API) qui semble être la remplaçante ?

Là je pinaille un peu :
- d'un point de vue purement esthétique, j'ai l'impression qu'il y a quelques problèmes de padding/margin, je l'ai notamment constaté sur les titres (quand on clique sur un titre dans la TOC, ça scrolle jusqu'à ce qu'il soit collé au bord haut de l'écran, ajouter un peu de padding serait plus joli)

De manière générale, on a quelques problèmes de rendu sur les TP, etc. mais je suis confiant sur le fait que ça ira mieux une fois que ces derniers auront été adaptés :wink: 

Globalement ça semble prometteur, il y a encore quelques détails à régler mais ça devrait rendre bien un fois fini !

**Comment by Nhqml** [comment]
Replaced by #76 (which has a cleaner git history)
